pipelines/templates/tasks/upmpackages.yml

```diff
@@ -0,0 +1,22 @@
+# [Template] Builds and publishes UPM .tgz packages
+parameters:
+  version: '$(MRTKVersion)-preview.$(Build.BuildNumber)'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '12.18.0' 
+- task: PowerShell@2
+  displayName: 'Build UPM packages'
+  inputs:
+    targetType: filePath
+    filePath: ./scripts/packaging/createupmpackages.ps1
+    arguments: >
+      -ProjectRoot ./
```

Comment:
I don't think it's good to take a dependency on the current working directory. Check out the other build scripts that get references to get specific directories on the build machine.

Reply:
CWD is hard to depend on...

Reply:
the script needs to know the project root in order to find the package.json files. this value gets resolved by the script and equates to the actual root (the build machine runs all scripts from the context of the project root (where Assets lives))

Reply:
i verified through several build machine tests that it starts in the actual project root and that this is the correct path to pass